src/minoversikt/moteplan/motekollonne.tsx

```diff
@@ -25,14 +60,22 @@ function MoteKollonne({dato, mote, enhetId}: MoteKollonneProps) {
 
             <Table.DataCell className="moteplan_tabell_deltaker">
                 {mote.deltaker.fnr && (
-                    <Link
-                        onClick={() => {
-                            setFraBrukerIUrl(mote.deltaker.fnr);
-                        }}
-                        href={getPersonUrl(mote.deltaker.fnr, '#visAktivitetsplanen', enhetId)}
-                    >
+                    <Button loading={laster} onClick={handterKlikk} size="xsmall" variant="tertiary">
                         {capitalize(mote.deltaker.etternavn)}, {capitalize(mote.deltaker.fornavn)}
-                    </Link>
+                    </Button>
+                )}
+                {mote.deltaker.fnr && harFeil && (
```

Comment:
How this can happen? If there is error with `modiacontextholder endpoint`? 

We have the same logic and more less the same error message in 3 places: `brukernavn.tsx`,  `sisteendringkategori.tsx`, `motekollonne.tsx`...so probably there is some way to reduce duplication.

Reply:
Yes, modiacontextholder could theoretically return an error, so we have to display this to the user. 

I tried thinking of ways to reduce the duplication even more than I already did, but it turned out to be quite hard. Also, removing duplication can often lead to making the code _less_ obvious/readable, depending on the context. When I tried to refactor this I felt that I ended up making the code _too_ clever, and I didn't like that. But there is probably other approaches one could take than what I did, som if you have some suggestions to how this could be refactored I'd be happy to hear them.